mobile/src/main/java/ch/epfl/sdp/mobile/ui/game/ar/ArChessBoardScreen.kt

```diff
@@ -68,22 +65,60 @@ fun <Piece : ChessBoardState.Piece> ArChessBoardScreen(
  * and change the displayed position
  *
  * @param arSceneView The view where the scene will be displayed
- * @param chessScene The scene that contains the boards node
+ * @param boardNode The node that will be added into the view
  * @param anchor The (new) board's anchor position
  */
-private fun <Piece : ChessBoardState.Piece> anchorOrMoveBoard(
-    arSceneView: ArSceneView,
-    chessScene: ChessScene<Piece>?,
+private fun anchorOrMoveBoard(
+    // arSceneView: ArSceneView,
+    boardNode: ArModelNode,
     anchor: Anchor
 ) {
 
-  val currentChessScene = chessScene ?: return
+  // FIXME : Workaround see line 55
```

Comment:
I don't think this is the right line number.

```suggestion
  // FIXME : Workaround see line 117
```

Reply:
Yeah, I forget to change it after my refactoring ^^"